README.md

```diff
@@ -98,84 +98,115 @@ This will pull and install CKAN and all it's dependencies into your virtual
 environment, create all necessary configuration files, launch external services
 using Docker Compose and start the CKAN development server.
 
-You can repeat the last command at any time to start developing again. 
+You can repeat the last command at any time to start developing again.
 
 Type `make help` to get a like of user commands useful to managing the local
-environment. 
+environment.
+
+Installing with Docker
+----------------------
+
+Unlike other CKAN extensions, external storage needs node modules to be installed
+and build in order to work properly. You will need to install node and npm.
+Below is how your Dockerfile might look like
+
+```
+RUN apt-get -q -y install \
+        python-pip \
+        curl \
+        git-core
+
+RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - && apt-get install nodejs && npm version
+
+# Install ckanext-external-storage
+RUN git clone --branch ${CKANEXT_EXTERNAL_STORAGE_VERSION} https://github.com/datopian/ckanext-external-storage
+RUN pip install --no-cache-dir -r "ckanext-external-storage/requirements.py2.txt"
+RUN pip install -e ckanext-external-storage
+
+# Install node modules and build the bundle
+RUN cd ckanext-external-storage && npm install && npm run build && cd -
+
+# Install other extensions
+...
+```
+
+__NOTE:__ We assume that you have Giftless server running with configuration as
```

Comment:
While I think this is OK, technically we need a Git LFS server, doesn't have to be Giftless. But Giftless is most likely the most suitable one (e.g. supporting the same auth schemes)